plugin/pkg/admission/persistentvolume/label/admission.go

```diff
@@ -75,6 +77,13 @@ func (l *persistentVolumeLabel) Admit(a admission.Attributes) (err error) {
 		}
 		volumeLabels = labels
 	}
+	if volume.Spec.GCEPersistentDisk != nil {
```

Comment:
Yes - this came up in review a few lines up ^^^.  I agree with the criticism, but it isn't clear to me how best to fix it.